release/202008/metrics-common-rc.md

```diff
@@ -0,0 +1,48 @@
+## Common Metrics
+Common Metrics Repository: https://github.com/chaoss/wg-common
+
+### Focus Area - What
+
+**Goal:**  
+Understand what contributions from organizations and people are being made.
+
+<div>
+<table>
+  <thead><tr><th>Metric/Details</th><th>Question</th><th>Issue</th></tr></thead>
+<tbody>
+  <tr><td><a href="https://github.com/chaoss/wg-common/blob/master/focus-areas/what/types-of-contributions.md">Types of Contributions [Under Review]</a></td><td>What types of contributions are being made?</td><td>[Issue \#56](https://github.com/chaoss/wg-common/issues/56)</td></tr>
```

Comment:
How about making the link to the issues a call to action?

This is an example:

```suggestion
  <thead><tr><th>Metric/Details</th><th>Question</th><th>Review</th></tr></thead>
<tbody>
  <tr><td><a href="https://github.com/chaoss/wg-common/blob/master/focus-areas/what/types-of-contributions.md">Types of Contributions [Under Review]</a></td><td>What types of contributions are being made?</td><td>[Provide Feedback](https://github.com/chaoss/wg-common/issues/56)</td></tr>
```

Reply:
I have been struggling with this one. If we use Provide Feedback, the exact link wording will be repeated in multiple rows. It makes a little more sense to me to have unique row info and have the column header provide the direction (Provide Feedback). Admittedly, the solution I have now isn't perfect and perhaps this isn't as big of a deal because we aren't opening comments for all the metrics like we have done in the past.